packages/react-core/src/components/Badge/Badge.tsx

```diff
@@ -7,6 +7,8 @@ export interface BadgeProps extends React.HTMLProps<HTMLSpanElement> {
   screenReaderText?: string;
   /**  Adds styling to the badge to indicate it has been read */
   isRead?: boolean;
+  /**  Adds styling to the badge to indicate it has a toggle */
+  isToggle?: boolean;
```

Comment:
For this, let's just add some verbiage to indicate that a badge w/ toggle must be passed to a MenuToggle component.

Reply:
Added the extra wording, done!